### DISCUSSION TITLE
Why Size of Jaeger-Span is so big

### DISCUSSION THREAD

**Discussion by winshenting** [jaegertracing/jaeger#3100] [Q&A]
Per day, my size of jaeger-span is too big. It cause the slowness of searching data.
green open qa-jaeger-span-2021-06-19           DopOAmcrQTCTg0yo1LlWpQ 5 1  681273925  0  57.5gb  28.8gb
green open qa-jaeger-span-2021-06-17           GmSnT2jBR3GiL2vwZvHjiQ 5 1 2082276707  0 161.9gb  80.9gb
However, Jaeger-Service and qa-jaeger-dependencies is KB. 
We have already use sampling policy
        param: 2.0
        type: ratelimiting
How could size of Span data is so big

**Comment by jpkrohling** [comment]
This really depends on what you are storing within your spans, as well as the number of spans you have stored... If you have concerns about the efficiency of some structures, we'd need more concrete information.
